docs/configuration.md

```diff
@@ -87,25 +87,45 @@ Supported placeholders/variables are:
  1. **{locale\_with\_underscore}** same as above except all hyphens '-' will be replaced with underscores '_'. This is typically used in properties and gettext projects.
  1. **{extension}** the source document file extension
 
-For example, given a source document is found at `/home/user/myproject/src/main/resource/message.properties` 
+For example, if your file structure is like following (where `{projectRoot}` is the root of your project and where zanata.xml lives):
```

Comment:
Suggest:
For example, if you have the following file structure (where `{projectRoot}` is the root directory of your project and contains zanata.xml):